dev/com.ibm.ws.wsoc/src/com/ibm/ws/wsoc/outbound/WsocOutboundChain.java

```diff
@@ -60,46 +63,97 @@ public class WsocOutboundChain {
     public static final String WS_CHAIN_NAME = "WsocOutboundHttp";
     public static final String WSS_CHAIN_NAME = "WsocOutboundHttpSecure";
 
-    public static VirtualConnection getVCFactory(WsocAddress addr) throws ChainException, ChannelException {
```

Comment:
Added details to https://github.com/OpenLiberty/open-liberty/issues/31332 to discuss why the vc factory is static implying that there would only be one possible outbound chain per server at a time